md-commons/src/main/java/com/wipro/ats/bdre/md/app/StoreJson.java

```diff
@@ -9,15 +9,15 @@
     private String name;
     private String id;
 
-    public List<AppValues> getAppValuesList() {
-        return appValuesList;
+    public List<AppValues> getColumns() {
+        return columns;
     }
 
-    public void setAppValuesList(List<AppValues> appValuesList) {
-        this.appValuesList = appValuesList;
+    public void setColumns(List<AppValues> columns) {
+        this.columns = columns;
     }
 
-    private List<AppValues> appValuesList;
+    private List<AppValues> columns;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)